### ISSUE TITLE
bazel failed to build due to libc mismatch

### ISSUE THREAD

**Issue by carlorosati** [Linuxbrew/homebrew-core#1498]
[```brew gist-logs bazel```](https://gist.github.com/d997abd40c016554c06843ad0b230f4c)

This most likely is due to jdk pointing to /lib64/libc.

```
% ldd `which java`
	linux-vdso.so.1 =>  (0x00007fffb5a11000)
	libpthread.so.0 => /lib64/libpthread.so.0 (0x00007f5c19757000)
	libjli.so => /local2/tmp/brew/Cellar/jdk/1.8.0-112/bin/../lib/amd64/jli/libjli.so (0x00007f5c19541000)
	libdl.so.2 => /lib64/libdl.so.2 (0x00007f5c1933d000)
	libc.so.6 => /lib64/libc.so.6 (0x00007f5c18fa9000)
	/lib64/ld-linux-x86-64.so.2 (0x00007f5c19974000)

```

I am working on installing openjdk. If successful, maybe we should replace the jdk package with openjdk.

**Comment by sjackman** [comment]
Is there a formula for `openjdk`?

**Comment by rwhogg** [comment]
> Is there a formula for openjdk?

[IcedTea](http://icedtea.classpath.org/wiki/Main_Page) might be a good thing to base it on. Fedora uses it; I don't remember if Debian does or not.

**Comment by sjackman** [comment]
> This most likely is due to jdk pointing to /lib64/libc.

A workaround is to run `brew patchelf jdk`.
A fix is to bottle `jdk`. I've opened PR https://github.com/Linuxbrew/homebrew-core/pull/1564 to build a bottle for `jdk`.

**Comment by sjackman** [comment]
Hopefully resolved by #1564. Please reopen this issue if it's not ultimately resolved by #1564.
